vue/src/TagmanagerTrackingCode/TrackingCodeCommon.vue

```diff
@@ -1,64 +1,93 @@
 <template>
   <div class="tagManagerTrackingCode">
-    <ActivityIndicator
-      v-show="isLoading"
-      :loading="true"
-      v-if="showContainerRow || environments.length > 1"
-    />
-    <div class="row"
-         v-if="showContainerRow || environments.length > 1"
-         v-show="!isLoading"
-    >
-      <div class="col s12 m4 ">
-        <div class="form-group row">
-          <div class="col s12 input-field">
-            <label
-              for="tagManagerTrackingCodeSite"
-              class="siteSelectorLabel"
-            >{{ translate('General_Website') }}</label>
-            <div class="sites_autocomplete">
-              <SiteSelector
-                id="tagManagerTrackingCodeSite"
-                v-model="site"
-                :show-all-sites-item="false"
-                :switch-site-on-select="false"
-                :show-selected-site="true"
-              />
-            </div>
-          </div>
+    <li v-if="showContainerRow">
+      {{ translate('TagManager_OptionallyCustomiseContainer') }}
+      <div class="trackingCodeAdvancedOptions">
+        <div class="advance-option">
+          <span>
+            <a href="javascript:;"
```

Comment:
Not sure if the `href="javascript:;"` serves any function when `@click.prevent` is present, but it probably doesn't hurt.

Reply:
Possibly not. I just copied it from another part of the code. It doesn't hurt anything.